apps/android/common/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -678,12 +678,14 @@
   <!-- markdown demo - MarkdownHelpView.kt -->
   <string name="how_to_use_markdown">How to use markdown</string>
   <string name="you_can_use_markdown_to_format_messages__prompt">You can use markdown to format messages:</string>
-  <string name="bold">bold</string>
-  <string name="italic">italic</string>
+  <string name="bold_font">bold</string>
```

Comment:
not all language files are renamed, same about the next one

Reply:
true